### PR TITLE
updated ua list

### PR DESCRIPTION
**Pull request by kermoshina**
Added URLs from the Presidential Order number 133/2017 that indicates a list of organizations to be sanctioned in Ukraine. Sanctions include blocking of websites. The full document and the list of websites is accessible here: http://www.president.gov.ua/storage/j-files-storage/00/40/38/6f76b8df9d0716da74bb4ae6a900d483_1494964345.pdf

### CODE REVIEW

**Comment by agrabeli** [comment]
@kermoshina awesome, thank you! :)

@hellais looks good, please merge.

File: lists/ua.csv [hunk 1]
```diff
@@ -941,3 +941,17 @@ http://lnr.today/,NEWS,News Media,2017-12-12,OONI,Included in blocklist: http://
 http://www.zapmeta.ws/,MMED,Media sharing,2017-12-12,OONI,Included in blocklist: http://inau.ua/sites/default/files/file/1711/vhidnyy_no_243_vid_28.11.2017.pdf
 https://lug24.ru/,NEWS,News Media,2017-12-12,OONI,Included in blocklist: http://inau.ua/sites/default/files/file/1711/vhidnyy_no_243_vid_28.11.2017.pdf
 https://www.rp-lnr.ru/,NEWS,News Media,2017-12-12,OONI,Included in blocklist: http://inau.ua/sites/default/files/file/1711/vhidnyy_no_243_vid_28.11.2017.pdf
+http://www.disk.yandex.ru,FILE,File-sharing,2017-12-16,oonitarian,sanction list
+http://www.translate.yandex.ru,SRCH,Search Engines,2017-12-16,oonitarian,sanction list
```
Comment (hellais) [inline lists/ua.csv]: The canonical URL of this is: https://translate.yandex.ru/

File: lists/ua.csv [hunk 2]
```diff
@@ -941,3 +941,17 @@ http://lnr.today/,NEWS,News Media,2017-12-12,OONI,Included in blocklist: http://
 http://www.zapmeta.ws/,MMED,Media sharing,2017-12-12,OONI,Included in blocklist: http://inau.ua/sites/default/files/file/1711/vhidnyy_no_243_vid_28.11.2017.pdf
 https://lug24.ru/,NEWS,News Media,2017-12-12,OONI,Included in blocklist: http://inau.ua/sites/default/files/file/1711/vhidnyy_no_243_vid_28.11.2017.pdf
 https://www.rp-lnr.ru/,NEWS,News Media,2017-12-12,OONI,Included in blocklist: http://inau.ua/sites/default/files/file/1711/vhidnyy_no_243_vid_28.11.2017.pdf
+http://www.disk.yandex.ru,FILE,File-sharing,2017-12-16,oonitarian,sanction list
```
Comment (hellais) [inline lists/ua.csv]: https://disk.yandex.ru/

File: lists/ua.csv [hunk 3]
```diff
@@ -941,3 +941,17 @@ http://lnr.today/,NEWS,News Media,2017-12-12,OONI,Included in blocklist: http://
...
+http://www.translate.yandex.ru,SRCH,Search Engines,2017-12-16,oonitarian,sanction list
+http://www.direct.yandex.ru/?from=all,COMM,E-commerce,2017-12-16,oonitarian,sanction list
```
Comment (hellais) [inline lists/ua.csv]: Duplicate

Comment (hellais) [inline lists/ua.csv]: https://direct.yandex.ru/

File: lists/ua.csv [hunk 4]
```diff
@@ -941,3 +941,17 @@ http://lnr.today/,NEWS,News Media,2017-12-12,OONI,Included in blocklist: http://
...
+http://drweb.ru,HACK,Hacking Tools,2017-12-16,oonitarian,sanctions list
```
Comment (hellais) [inline lists/ua.csv]: https://www.drweb.ru/

**Comment by hellais** [comment]
@agrabeli can you look at the URLs and write down the correct canonical URL for each of them? Many of them are off.

**Comment by agrabeli** [comment]
@kermoshina It's important that URLs are added in their canonical form, exactly as they would appear when accessing them via a web browser. Minor details (e.g. adding a URL on HTTP rather than HTTPS, or including www when that's not in the canonical URL) can lead to inaccurate measurements, which is why it's important that a URL is identical to what it would look like when accessed from a web browser. Apologies for the hassle, and thanks for your contributions!

@hellais I don't have permissions to edit the file directly. I've provided the right URLs in comments --- if you could use them to edit directly and then merge, that would be awesome, thank you.

**Comment by hellais** [comment]
Ha, normalising them lead us to detect a duplicate. Good thing we did it. I will remove it and hopefully then merge.

**Pull request merged**